### PR TITLE
Initial Ruleset

### PR DESCRIPTION
**Pull request by kunzel-andrew**
Signed-off-by: Andrew Kunzel <kunzel@fb.com>

### CODE REVIEW

**Review by cmiych — APPROVED** [review]

File: mutable_rules.txt [hunk 1]
```diff
@@ -0,0 +1,47 @@
+201. Players shall alternate in clockwise order, taking one whole turn apiece. Turns may not be skipped or passed, and parts of turns may not be omitted. All players begin with zero points.
```
Comment (tgilino) [inline mutable_rules.txt:1]: should be "alphabetical by last name, first name"?

**Review by tgilino — APPROVED** [review]

**Review by tgilino — DISMISSED** [review]

**Review by cmiych — DISMISSED** [review]

**Review by rookswr — CHANGES_REQUESTED** [review]
I think the note associated with rule 103 should be included.  Ran a diffcheck and everything else looks good.

**Comment by kunzel-andrew** [comment]
> I think the note associated with rule 103 should be included. Ran a diffcheck and everything else looks good.

Which note associated with Rule 103?

**Comment by kunzel-andrew** [comment]
@rookswr I figured out what you meant and I added the note. Please feel free to approve this initial PR.

**Review by tgilino — APPROVED** [review]

**Review by rookswr — APPROVED** [review]

**Comment by cmiych** [comment]
I am here, but unclear on what action is required of me.

**Pull request merged**